server/controller/routes/postAnswers.js

```diff
@@ -0,0 +1,26 @@
+const addAnswer = require("../../db/queries/postAnswers");
+
+module.exports = (req, res, next) => {
+  try {
+    // this is if we send all the answers as a object in the post body
+    // the structure needs to be { answer1, answer2, answer3 } = req.body.data;
+
+    const preValidatedAnswers = req.body.data;
+
+    addAnswer(preValidatedAnswers).then(result => {
+      // addAnswers returns the id which is auto generated int - this means the SQL database hase processed correctly
+      if (Number.isInteger(result.rows[0].id)) {
+        res.send({
+          data: "ok",
+          statusCode: 200
+        });
+      } else
+        throw res.send({
```

Comment:
so, i think you have a misunderstanding for the functionality of throw, 
so you have read about that. 
so your code will make two responses.
```suggestion
        throw new Error('there is an error in validation ');

Reply:
ahh your right

Reply:
I think the validation will throw an error at a different point this will be in regards addAnsmer